build.sbt

```diff
@@ -167,45 +203,46 @@ lazy val `play-json` = crossProject(JVMPlatform, JSPlatform)
           .map {
             i =>
               def commaSeparated(s: Int => String)   = 1.to(i).map(s).mkString(", ")
-              def newlineSeparated(s: Int => String) = 1.to(i).map(s).mkString("\n")
+              def newlineSeparated(s: Int => String) = 1.to(i).map(s).mkString("\n        ")
```

Comment:
This fixes the indentation of the generated file, which generates a warning under Scala 3.  Not a big deal, but annoying when you have to clean/compile 1000 times (because `inline def`s don't work well with the incremental compiler).